devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1137,8 +1183,8 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     std::string estimateJointVelocityAccelerationOptionName = "estimateJointVelocityAcceleration";
     if( !(prop.check(estimateJointVelocityAccelerationOptionName.c_str()) && prop.find(estimateJointVelocityAccelerationOptionName.c_str()).isBool()) )
     {
-        yWarning() << "wholeBodyDynamics: estimateVelocityAccelerationOptionName bool parameter missing, please specify it.";
-        yWarning() << "wholeBodyDynamics: setting estimateVelocityAccelerationOptionName to the default value of true, but this is a deprecated behaviour that will be removed in the future.";
+        yWarning() << "wholeBodyDynamics: estimateJointVelocityAcceleration bool parameter missing, please specify it.";
+        yWarning() << "wholeBodyDynamics: setting estimateJointVelocityAcceleration to the default value of true, but this is a deprecated behaviour that will be removed in the future.";
```

Comment:
```suggestion
        yWarning() << "wholeBodyDynamics: setting estimateJointVelocityAcceleration to the default value of false, but this is a deprecated behavior that will be removed in the future.";
```

Reply:
Ah cool, so there is no problem in https://github.com/robotology/robots-configuration/issues/668 .